pkg/ocm/client.go

```diff
@@ -124,6 +124,7 @@ func (b *ClientBuilder) Build() (result *Client, err error) {
 	if b.cfg.URL != "" {
 		builder.URL(b.cfg.URL)
 	}
+	builder.AlternativeURL("/api/accounts_mgmt", "https://api.integration.openshift.com")
```

Comment:
Please remove this.

Reply:
The cluster creation does not work without this. That's why I opened the MR to ask for feedback on how to resolve this issue.

Reply:
Maybe create a proxy in your localhost, or some /etc/hosts magic? I think some other folks that don't use `crc` use nginx for proxying. I've never done that, since `crc` handles all routing natively without changes in the code. Alternatively you can deploy AMS locally.